orm/ai-sdk-nextjs/README.md

```
@@ -0,0 +1,94 @@
+
+That's it, your project is now configured to use Prisma Postgres!
+
+### 3. Generate and migrate Prisma client
+
+1. Run the following command to generate the Prisma Client. This is what you will be using to interact with your database.
+
+```
+npx prisma generate
+```
+
+2. Migrate the DB
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)